docs/docs/tutorial-gce.md

```diff
@@ -1,6 +1,11 @@
 ---
 title: Mesos-DNS on GCP
+
+redirect_to:
+- http://mesosphere.github.io/mesos-dns/
+
 ---
+<!-- This is deprecated -->
```

Comment:
@jdef There's talk of possibly rewriting the tuturial, without GCE - so left in. Feel free to remove.